src/stores/ProjectsInstitutionStore.js

```diff
@@ -48,25 +61,34 @@ export const useProjectInstitutionStore = defineStore({
       }
     },
 
-    async addInstitution(projectId, institutionId) {
+    async addInstitution(projectId, institutionId, name) {
       const url = `${
         import.meta.env.VITE_API_URL
-      }/projects/${projectId}/institutions/create`
+      }/projects/${projectId}/institutions/add`
 
-      const response = await axios.post(url, { institutionId })
+      const response = await axios.post(url, { institutionId, name })
 
       if (response.status == 200) {
         const institution = response.data.institution
 
         this.institutions.push({
-          institutionId: institution.institution_id,
+          institution_id: institution.institution_id,
           name: institution.name,
         })
 
-        return institution
+        return true
       }
-
       return false
     },
+
+    async editInstitution(projectId, institutionId, name) {
+      const url = `${
+        import.meta.env.VITE_API_URL
+      }/projects/${projectId}/institutions/edit`
+
+      const response = await axios.post(url, { institutionId, name })
```

Comment:
After an edit to an institution, you must update the `institutions` field so that the project's institutions are up-to-date. This ensures that all uses of `institutions` are updated immediately.